app/controllers/application_controller.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::API
+  include DeviseTokenAuth::Concerns::SetUserByToken
```

Comment:
WDYTA creating an `ApiController` class and include this there? The reason for this is that we don't want this authentication method for non-api responses